FinSearchUnified/Tests/BusinessLogic/Models/FindologicArticleModelTest.php

```diff
@@ -1571,4 +1571,81 @@ public function testPseudoSalesAreExportedCorrectly($pseudoSales, $expectedSales
             $values->getValues()
         );
     }
+
+    public function testCoverImageIsExportedAsFirstImage()
+    {
+        $articleConfiguration = [
+            'name' => 'FindologicArticle 1',
+            'active' => true,
+            'tax' => 19,
+            'supplier' => 'Findologic',
+            'categories' => [
+                ['id' => 3],
+                ['id' => 5]
+            ],
+            'images' => [
+                [
+                    "mediaId" => 2, // Muensterlaender_Lagerkorn_Ballons_Hochformat
+                    'main' => 0,
+                    'position' => 1,
+                ],
+                [
+                    "mediaId" => 3, // Muensterlaender_Lagerkorn_Imagefoto
+                    'main' => 1,
+                    'position' => 2,
+                ],
+                [
+                    "mediaId" => 4, // Muensterlaender_Lagerkorn_Produktion
+                    'main' => 0,
+                    'position' => 3,
+                ],
+            ],
+            'mainDetail' => [
+                'number' => 'FINDOLOGIC1',
+                'active' => true,
+                'inStock' => 16,
+                'prices' => [
+                    [
+                        'customerGroupKey' => 'EK',
+                        'price' => 99.34,
+                    ],
+                ]
+            ],
+        ];
+
+        $articleFromConfiguration = $this->createTestProduct($articleConfiguration);
+
+        $baseCategory = new Category();
+        $baseCategory->setId(1);
+
+        $findologicArticle = $this->articleFactory->create(
+            $articleFromConfiguration,
+            'ABCD0815',
+            [],
+            [],
+            $baseCategory
+        );
+
+        $xmlArticle = $findologicArticle->getXmlRepresentation();
+
+        $reflector = new ReflectionClass(Item::class);
+        $imagesProperty = $reflector->getProperty('images');
+        $imagesProperty->setAccessible(true);
+        $images = $imagesProperty->getValue($xmlArticle);
+        $images = array_pop($images);
+
+        $this->assertEquals(
```

Comment:
Maybe you want to add an `$expectedCoverImageUrl` at the start, and explicitly assert that this is on the first place. Don't have a really strong opinion on this, but this would make the test a bit more clear from the beginning.